src/main/java/com/redhat/coolstore/service/ShoppingCartOrderProcessor.java

```diff
@@ -1,35 +1,29 @@
 package com.redhat.coolstore.service;
 
 import java.util.logging.Logger;
-import javax.ejb.Stateless;
-import javax.annotation.Resource;
-import javax.inject.Inject;
-import javax.jms.JMSContext;
-import javax.jms.Topic;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import io.smallrye.reactive.messaging.Channel;
```

Comment:
I do notice that io.smallrye is not added to the pom, are you still getting mvn compile time errors?

Reply:
@shawn-hurley so there is an issue about maven parsing I am fixing now. The issue is that - we add a bad quarkus dip but we don't fix that because of parsing issues and then from that point onwards, we always get the same error in pom ignoring everything else.